docs/api/README.md

```diff
@@ -75,6 +81,8 @@ LTS (Focal Fossa), then we would look for the artifact at:
 
 `http://{kf repo configured in l3afd.cfg}/ratelimiting/latest/focal/l3af_ratelimiting.tar.gz`
 
+If `object_file` and `entry_function_name` are not provided in the payload, then bpf programs are not loaded and chaining is not allowed.
```

Comment:
In lines 52-73 they're "eBPF programs".  Here they're "bpf programs".  Be consistent.
I'd prefer "BPF programs".